includes/Handlers/Connection.php

```diff
@@ -255,7 +261,42 @@ public function get_redirect_url() {
 	 */
 	public function get_connect_parameters() {
 
-		return [];
+		return [
+			'client_id'     => self::CLIENT_ID,
+			'redirect_uri'  => self::PROXY_URL,
+			'display'       => 'page',
+			'response_type' => 'token',
+			'scope'         => $this->get_scopes(),
```

Comment:
by looking a the Authentication doc from FB I'm not sure if this can be an array of strings - the doc only mentions "string", does it mean comma separated strings? json encoded?

Reply:
never mind it looks like it has to be comma separated

Reply:
see https://github.com/facebookincubator/facebook-for-woocommerce/pull/1287/commits/e561825e436c75730bfe2ac22876f0b7bdca576e